_rules/css-restrict-orientation-b33eff.md

```diff
@@ -0,0 +1,258 @@
+---
+id: b33eff
+name: Page is not restricted to either `landscape` or `portrait` orientation using CSS transform property
+rule_type: atomic
+description: |
+  This rule checks that page content is not restricted to either `landscape` or `portrait` orientation using CSS transform property.
+accessibility_requirements:
+  wcag20:1.3.4: # Orientation
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'title-require'
+---
+
+## Applicability
+
+The rule applies to any element that is [visible](#visible) and has a CSS [transform](https://www.w3.org/TR/css-transforms/#propdef-transform) property with any of the below [transformation functions](https://www.w3.org/TR/css-transforms/#transform-functions)
```

Comment:
"any element" is pretty open ended. Should we limit this to any HTML and SVG element? Also definitely shouldn't only have HTML test cases, unless you limit this rule to HTML elements only (which you could also do).

Reply:
I'm thinking that limiting this to HTML would make sense.